hive-event-listeners/apiary-gluesync-listener/src/test/java/com/expediagroup/apiary/extensions/gluesync/listener/IcebergTableOperations.java

```diff
@@ -0,0 +1,119 @@
+package com.expediagroup.apiary.extensions.gluesync.listener;
+
+import static org.apache.iceberg.BaseMetastoreTableOperations.ICEBERG_TABLE_TYPE_VALUE;
+import static org.apache.iceberg.BaseMetastoreTableOperations.TABLE_TYPE_PROP;
+import static org.apache.iceberg.PartitionSpec.builderFor;
+import static org.apache.iceberg.types.Types.NestedField.required;
+
+import java.util.Collections;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Optional;
+import java.util.UUID;
+
+import org.apache.hadoop.hive.metastore.TableType;
+import org.apache.hadoop.hive.metastore.api.SerDeInfo;
+import org.apache.hadoop.hive.metastore.api.StorageDescriptor;
+import org.apache.hadoop.hive.metastore.api.Table;
+import org.apache.hadoop.hive.metastore.api.hive_metastoreConstants;
+import org.apache.iceberg.PartitionSpec;
+import org.apache.iceberg.PartitionSpecParser;
+import org.apache.iceberg.Schema;
+import org.apache.iceberg.SchemaParser;
+import org.apache.iceberg.SortOrder;
+import org.apache.iceberg.SortOrderParser;
+import org.apache.iceberg.TableProperties;
+import org.apache.iceberg.hive.HiveSchemaUtil;
+import org.apache.iceberg.types.Types;
+
+import com.google.common.collect.Maps;
+
+public class IcebergTableOperations {
+  protected static Table simpleIcebergTable(String database, String tableName, Schema schema, PartitionSpec partitionSpec,
```

Comment:
Minor. you can create another function: 

`simpleIcebergTable(String database, String tableName, Schema schema, PartitionSpec partitionSpec,
 
Add a suggestion, <Cmd+g>
Add heading textAdd bold text, <Cmd+b>Add italic text, <Cmd+i>
Add a quote, <Cmd+Shift+.>Add code, <Cmd+e>Add a link, <Cmd+k>
Add a bulleted list, <Cmd+Shift+8>Add a numbered list, <Cmd+Shift+7>Add a task list, <Cmd+Shift+l>
Directly mention a user or team
Reference an issue, pull request, or discussion
Add saved reply
Minor. you can create another function: 

`simpleIcebergTable(String database, String tableName, SortOrder sortOrder)`

where the function `simpleIcebergSchema` and `simpleIcebergPartitionSpec` is called internally and then in your test, you don't have to have to import them.

Reply:
In reality I did it on purpose, in case that you want to add other tests where you actually want to change the partitioning for example, and have a specific test partitioning.

Reply:
For example, make testing when adding 2 partitioning functions or maybe none.

Reply:
Yes. When those tests are added, then those could be imported.

Not a biggie, I have approved already.

Reply:
Probably the solution was overloading, I will keep an eye in my next contrib